lib/src/datastore_impl.dart

```diff
@@ -267,7 +267,7 @@ class DatastoreImpl implements datastore.Datastore {
     return orders.map(_convertDatastore2ApiOrder).toList();
   }
 
-  static Future _handleError(error, stack) {
+  static FutureOr<Null> _handleError(error, stack) {
```

Comment:
Why `FutureOr`? This function is guaranteed to return a `Future`?

Reply:
Yeah. The `<Null>` was the important bit. Fixed.